### PR TITLE
made the game introduction 

### PR DESCRIPTION
**Pull request by StickWolf**

### CODE REVIEW

**Review by XanderSuFromQBIX — APPROVED** [review]
### 

> _**I, Will Approve!**_

File: GameEngine/Engine.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace GameEngine
 {
     public class Engine
     {
-        private List<Character> AllCharacters { get; set; }
+        private List<Character> AllCharacters { get; set; } = new List<Character>();
```
Comment (Materia-xx) [inline GameEngine/Engine.cs:11]: Nice fix!

**Review by Materia-xx — APPROVED** [review]
Good work.

**Comment by XanderSuFromQBIX** [comment]
I hope you did it the right way,
and that  _**"I"**_ helped you with the right things at the time.

**Pull request merged**